.github/workflows/ci.yml

```diff
@@ -45,6 +45,10 @@ jobs:
         run: isort . --check
         if: success() || failure()
 
+      - name: Check mypy formatting
```

Comment:
do we need another linter?

Reply:
this is just for type hint, this was suggested 2 weeks ago.
Unfortunately black and flake8 don't provide type hint checks.